### ISSUE TITLE
Assigning an energy gas usage rate breaks the config page

### ISSUE THREAD

**Issue by tinglis1** [home-assistant/frontend#9942]
### The problem

When assigning a gas usage rate the energy config page is no longer available and just has the link “go back”. 

If I go into the .storage energy file, remove the usage rate and restart au can access the config page again.

My usage rate is a template sensor with the unit AUD/m3. I initiate set it to the wrong entity of an input_number with a unit of $/MJ and got the same problem. I thought it was due to that initially but the correct sensor with correct units didn’t fix the issue. 

My install is a VM of home assistant OS on unraid. 

### What is version of Home Assistant Core has the issue?

2021.9.1

### What type of installation are you running?

Home Assistant OS

### Integration causing the issue

Energy dashboard

### Anything in the logs that might be useful for us?

```txt
Nothing in the logs that I could see was related to this error
```

**Comment by balloob** [comment]
Can you look at your browser console for any errors? Or look in your logs ?

**Comment by tinglis1** [comment]
Issue seems to have been resolved in 2021.9.5.

**Comment by emontnemery** [comment]
Thanks for reporting back.
This was probably solved by https://github.com/home-assistant/core/pull/56057
